package.json

```diff
@@ -185,7 +184,7 @@
     "stream-browserify": "^3.0.0",
     "stream-http": "^3.2.0",
     "styled-components": "^5.3.6",
-    "wagmi": "^0.6.5",
+    "wagmi": "0.12.1-cjs",
```

Comment:
Interesting this even works but looking at the links you posted in https://github.com/shapeshift/web/pull/4157#issuecomment-1489590242, looks sane, nice find ser 🤌